src/main/java/com/google/api/codegen/transformer/PackageMetadataTransformer.java

```diff
@@ -27,6 +34,16 @@
       String template,
       String outputPath,
       TargetLanguage language) {
+    return generateMetadataView(packageConfig, model, template, outputPath, language, null);
+  }
+
+  public PackageMetadataView.Builder generateMetadataView(
+      PackageMetadataConfig packageConfig,
+      Model model,
+      String template,
+      String outputPath,
+      TargetLanguage language,
+      Set<String> whitelistedDependencies) {
```

Comment:
It is not very clear that what this `whitelistedDependencies` does. Add some comments.